### ISSUE TITLE
Full screen carousel

### ISSUE THREAD

**Issue by baj84** [OwlCarousel2/OwlCarousel2#2553]
Is it possible to make the individual slides 100% width and height of the screen?

**Comment by easterneas** [comment]
You mean slide-by-page slider? If you mean by that, I was looking for the same feature, too.

**Comment by baj84** [comment]
Yes, but with a single slide being 100% width and height of the browser window

**Comment by KeySeeDev** [comment]
Yeah, you should make every parent 100% height the width is already there.
```
.owl-carousel.fullh
.owl-carousel.fullh .owl-stage-outer,
.owl-carousel.fullh .owl-stage,
.owl-carousel.fullh .owl-item {height: 100%}
```
for width make sure to set in options `{items: 1}`
